build.gradle

```diff
@@ -6,7 +6,7 @@ buildscript {
 project.ext.moduleName = 'com.synopsys.integration.jenkins-common'
 project.ext.javaUseAutoModuleName = 'true'
 
-version = '0.5.4-SNAPSHOT'
+version = '0.5.4'
```

Comment:
release build handles versioning. Without the -SNAPSHOT, release build will fail

Reply:
That would be bad... I had forgotten that detail... Thanks!